scripts/template.py

```diff
@@ -0,0 +1,48 @@
+"""
```

Comment:
Could make it a bit more explicit that this is acting on Jinja2 templates?

Like this approach though, bit less hacky than mine (using `tr` iirc...)